app/move/controllers/create/move-details.js

```diff
@@ -9,30 +10,24 @@ const referenceDataHelpers = require('../../../../common/helpers/reference-data'
 class MoveDetailsController extends CreateBaseController {
   async configure(req, res, next) {
     try {
-      const courts = await referenceDataService.getLocations({
-        type: 'court',
+      const courtLocations = await referenceDataService.getLocations({
+        filter: {
+          'filter[location_type]': 'court',
```

Comment:
I’d be tempted to create a method in the service like we talked about to abstract this logic. 

This syntax is specific to the API so it would be better to have it contained within one service rather than living in this controller.

Reply:
Totally agreed I wanted to get this simple fix up so I can finish the sidebar work and then do the abstraction we spoke about in a separate PR connected with the sidebar work.